### PR TITLE
docs: fix explanation of Injectable decorator in architecture guide

### PR DESCRIPTION
**Pull request by stevermeister**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## Other information

it was wrong explanation of @Injectable decorator

### CODE REVIEW

**Comment by stevermeister** [comment]
hm, it says problem with commit message, what's wrong with it?
https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit

**Comment by alfaproject** [comment]
`fix` is not a valid scope. That's a commit type, but in here the type is `docs`.

This is what you probably want: `docs: fix wrong explanation of @Injectable decorator`

File: aio/content/guide/architecture.md [hunk 1]
```diff
@@ -73,7 +73,7 @@ Angular provides predefined pipes for common transformations, and you can also d
 
 ## Services and dependency injection
 
-For data or logic that isn't associated with a specific view, and that you want to share across components, you create a *service* class. A service class definition is immediately preceded by the `@Injectable()` decorator. The decorator provides the metadata that allows your service to be *injected* into client components as a dependency.
+For data or logic that isn't associated with a specific view, and that you want to share across components, you create a *service* class. A service class definition is immediately preceded by the `@Injectable()` decorator.  The decorator provides the metadata that allows to *inject* another services into your service as a dependency.
```
Comment (trotyl) [inline aio/content/guide/architecture.md]: The requirement only applies to [`ClassProvider`](https://angular.io/api/core/ClassProvider). For example, [`Router`](https://angular.io/api/router/Router) is a well-known service that has many dependencies, but not having `Injectable()` due to it's created by [`FactoryProvider`](https://angular.io/api/core/FactoryProvider):

https://github.com/angular/angular/blob/8282e15c2becbe42a49befa07d6407247e8243d8/packages/router/src/router.ts#L226-L228

Comment (stevermeister) [reply aio/content/guide/architecture.md]: you are right, but current version is completely wrong

Comment (brandonroberts) [reply aio/content/guide/architecture.md]: I'm not sure how this is more clear. It doesn't enable you to inject services, it enables your service to be injected somewhere else. How about:

`The Injectable decorator provides metadata that allows your class provider needs to be *injected* as a dependency.`

Comment (trotyl) [reply aio/content/guide/architecture.md]: @brandonroberts That's the wrong explanation today, when `MyService` being decorated with `@Injectable()`, it **means** other dependencies can be injected into `MyService`. It **doesn't mean** `MyService` can be injected into other types.

Comment (brandonroberts) [reply aio/content/guide/architecture.md]: Gotcha. The statement still could be clearer.

`
The decorator provides the metadata that allows other providers to be **injected** into your service.
`

Comment (trotyl) [reply aio/content/guide/architecture.md]: `Service` is a common case, but not the only one, for example [a base class of component/directive also need `@injectable()`](https://github.com/angular/angular/issues/16566#issuecomment-337960028).

I'd prefer a more general wording that fits all conditions.

Comment (brandonroberts) [reply aio/content/guide/architecture.md]: More general as in `dependencies` instead of `providers`?

Comment (trotyl) [reply aio/content/guide/architecture.md]: I mean `into your type/class` rather than `into your service`, as the target might not be a service.

Comment (stevermeister) [reply aio/content/guide/architecture.md]: @brandonroberts , 
yes thx. hope I understand you correctly

Comment (trotyl) [reply aio/content/guide/architecture.md]: I didn't really mean `type/class` literally, but whichever of them... 😢

File: aio/content/guide/architecture.md [hunk 2]
```diff
@@ -73,7 +73,7 @@ Angular provides predefined pipes for common transformations, and you can also d
 
 ## Services and dependency injection
 
-For data or logic that isn't associated with a specific view, and that you want to share across components, you create a *service* class. A service class definition is immediately preceded by the `@Injectable()` decorator. The decorator provides the metadata that allows your service to be *injected* into client components as a dependency.
+For data or logic that isn't associated with a specific view, and that you want to share across components, you create a *service* class. A service class definition is immediately preceded by the `@Injectable()` decorator.  The decorator provides the metadata that allows to *inject* another services into your type/class as a dependency.
```
Comment (brandonroberts) [inline aio/content/guide/architecture.md]: Slight tweak.

`
The decorator provides the metadata that allows other providers to be **injected** into your type/class as a dependency.
`

**Review by brandonroberts — CHANGES_REQUESTED** [review]

File: aio/content/guide/architecture.md (hunk 1 above)
Comment (stevermeister) [reply aio/content/guide/architecture.md]: my bad, fixed

File: aio/content/guide/architecture.md (hunk 2 above)
Comment (stevermeister) [reply aio/content/guide/architecture.md]: is it good now?

File: aio/content/guide/architecture.md [hunk 3]
```diff
@@ -73,7 +73,7 @@ Angular provides predefined pipes for common transformations, and you can also d
 
 ## Services and dependency injection
 
-For data or logic that isn't associated with a specific view, and that you want to share across components, you create a *service* class. A service class definition is immediately preceded by the `@Injectable()` decorator. The decorator provides the metadata that allows your service to be *injected* into client components as a dependency.
+For data or logic that isn't associated with a specific view, and that you want to share across components, you create a *service* class. A service class definition is immediately preceded by the `@Injectable()` decorator. The decorator provides the metadata that allows to *inject* another services into your class as a dependency.
```
Comment (brandonroberts) [inline aio/content/guide/architecture.md]: Should read as:

`The decorator provides the metadata that allows other providers to be **injected** into your class as a dependency.`

**Review by brandonroberts — CHANGES_REQUESTED** [review]
The decorator provides the metadata that allows to *inject* another services into your class as a dependency.

**Review by brandonroberts — CHANGES_REQUESTED** [review]

**Comment by stevermeister** [comment]
@brandonroberts sure, will do

**Comment by stevermeister** [comment]
@brandonroberts done

**Review by brandonroberts — APPROVED** [review]

**Comment by brandonroberts** [comment]
Caretaker note: This is a docs-only change and I am using a global approval.

**Comment by brandonroberts** [comment]
Thanks @stevermeister!

**Pull request closed without merging**